openwisp_notifications/handlers.py

```diff
@@ -195,13 +196,16 @@ def send_email_notification(sender, instance, created, **kwargs):
     else:
         target_url = None
     if target_url:
-        description += '\n\nFor more information see {0}.'.format(target_url)
+        description += _('\n\nFor more information see {0}.'.format(target_url))
```

Comment:
This won't work for the reasons expressed here: https://github.com/openwisp/openwisp-notifications/pull/208#discussion_r741145797

Reply:
>Python’s str.format() method will not work when either the format_string or any of the arguments to str.format() contains lazy translation objects. Instead, you can use django.utils.text.format_lazy(), which creates a lazy object that runs the str.format() method only when the result is included in a string. 

This is what the doc mentions. str.format doesn't work when there are lazy objects. But, I don't think we are translating `target_url` too. :thinking:

Reply:
That's a parameter that has to be passed, pelase read the django docs.